src/rest_api/api/compound_processing/standardize.rs

```diff
@@ -3,12 +3,20 @@ use crate::{rest_api::models::Smiles, search::compound_processing::standardize_s
 use poem_openapi::payload::Json;
 use rayon::prelude::*;
 
-pub async fn v1_standardize(mol: Json<Vec<Smiles>>) -> StandardizeResponse {
+pub async fn v1_standardize(
+    mol: Json<Vec<Smiles>>,
+    attempt_fix: Option<&str>,
+) -> StandardizeResponse {
+    let attempt_fix = match attempt_fix {
+        Some("true") => true,
```

Comment:
```
Some(_) => true,
```

so we would support `?attempt_fix=wizbang` and even `?attempt_fix` which implies `""`.